src/animals/Dog.java

```diff
@@ -1,11 +1,14 @@
 package animals;
 
 import java.util.Random;
-import veterinary.Vet;
 
 public class Dog extends Animal {
 	
-    public DogBreed dogbreed;
+  /**
+   * DogBreed enum dogBreed is defined for the Dog Class. It is used in the Dog constructor.
+   * dogName is a string array which contains the list of possible names for dogs.
+   */
+  public DogBreed dogBreed;
 	public static String[] dogName = {
```

Comment:
tabbing and same comment as in cat where the comment needs to be located above the relevant code.

Reply:
fixed as in the Cats doc